reverse-strings/tools/reverse_test.go

```diff
@@ -0,0 +1,30 @@
+package tools
+
+import (
+	"testing"
+)
+
+// TestReverse tests the string reverse method implementation in a multiple cases.
+func TestReverse(t *testing.T) {
+	tests := map[string]struct {
+		input  string
+		expRes string
+	}{
+		"AHCII 7-bit chars":     {input: "aBcDeF12", expRes: "21FeDcBa"},
+		"Russian chars":         {input: "РазРаботкА", expRes: "АктобаРзаР"},
+		"Separators":            {input: ".,/ ", expRes: " /,."},
+		"Chinese chars":         {input: "  中华人民共和国 ", expRes: " 国和共民人华中  "},
+		"Special Unicode chars": {input: "✠⚛✪⚡", expRes: "⚡✪⚛✠"},
+		"Diacritical chars":     {input: "ỗỆἣ", expRes: "ἣỆỗ"},
+		"":                      {input: "", expRes: ""},
```

Comment:
Test name is empty but it must describe what exactly it tests
for instance - empty string